packages/core/parcel-bundler/src/assets/MarkdownAsset.js

```diff
@@ -0,0 +1,17 @@
+const localRequire = require('../utils/localRequire');
+const Asset = require('../Asset');
+
+class MarkdownAsset extends Asset {
+  constructor(name, options) {
+    super(name, options);
+    this.type = 'html';
+    this.needsPipelineProcessing = true;
+    this.hmrPageReload = true;
+  }
+  async parse(code) {
+    let marked = await localRequire('marked', this.name);
+    this.contents = marked(code);
```

Comment:
Rather than reassigning `this.contents`, you could leave it as the original code and just return the result of `marked` from the `generate` method instead of `parse`.

Reply:
Thanks!

Please, see the latest version.